devel/ci/integration/tests/test_bodhi.py

```diff
@@ -632,3 +633,302 @@ def test_get_builds_json(bodhi_container, db_container):
         with read_file(bodhi_container, "/httpdir/errorlog") as log:
             print(log.read())
         raise
+<<<<<<< HEAD
```

Comment:
This shouldn't be here

Reply:
yup, we need to edit/fix this backport

Reply:
Working on it

Reply:
Ok, the fix for `test_get_compose_view()` is failing in 4.1, maybe this was caused by some PR, that is not yet in 4.1

Reply:
Should I remove the fix

Reply:
With the latest push, this should be resolved.